### PR TITLE
Add new model to HuggingChat config

### PR DESCRIPTION
**Pull request by nsarrazin**
Tested the template with system prompt and without, seems to work well. It's quite fast already 🚀 

This specific config will work as soon as the 32k context window is up

### CODE REVIEW

**Comment by lewtun** [comment]
FYI the chat template should be identical to zephyr-7b-beta

**Comment by nsarrazin** [comment]
@lewtun We use the jinja template from the `https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1/blob/main/tokenizer_config.json#L32`directly! Seems to work well, let me know if we should use another one 👀

File: .env.template [hunk 1]
```diff
@@ -27,6 +27,30 @@ MODELS=`[
...
+    "promptExamples" : [
+      {
+        "title": "Write an email from bullet list",
+        "prompt": "As a restaurant owner, write a professional email to the supplier to get these products every week: \n\n- Wine (x10)\n- Eggs (x24)\n- Bread (x12)"
+      }, {
+        "title": "Code a snake game",
+        "prompt": "Code a basic snake game in python, give explanations for each step."
+      }, {
+        "title": "Assist in a task",
+        "prompt": "How do I make a delicious lemon cheesecake?"
```
Comment (lewtun) [inline .env.template:50]: This is a pretty impressive example from @osanseviero 

```suggestion
        "prompt": "Kayley has three brothers. Each of her brothers has two sisters. How many sisters does Kayley have?"
```

File: .env.template [hunk 2]
```diff
@@ -27,6 +27,30 @@ MODELS=`[
...
+    },
+    "promptExamples" : [
+      {
+        "title": "Write an email from bullet list",
+        "prompt": "As a restaurant owner, write a professional email to the supplier to get these products every week: \n\n- Wine (x10)\n- Eggs (x24)\n- Bread (x12)"
+      }, {
+        "title": "Code a snake game",
+        "prompt": "Code a basic snake game in python, give explanations for each step."
+      }, {
+        "title": "Assist in a task",
```
Comment (lewtun) [inline .env.template:49]:
```suggestion
        "title": "Logical reasoning",
```

**Review by lewtun — COMMENTED** [review]
Left some suggestions for the prompts

**Comment by lewtun** [comment]
> @lewtun We use the jinja template from the [tokenizer config](https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1/blob/main/tokenizer_config.json#L32) directly! Seems to work well, let me know if we should use another one 👀

No that's perfect!

File: .env.template [hunk 3]
```diff
@@ -27,6 +27,30 @@ MODELS=`[
...
+    "modelUrl": "https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "websiteUrl": "https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "logoUrl": "https://huggingface.co/datasets/huggingchat/models-logo/resolve/main/zephyr-logo.png",
+    "parameters": {
+      "truncate" : 24576,
+      "max_new_tokens" : 8192,
+    },
+    "promptExamples" : [
+      {
+        "title": "Write an email from bullet list",
```
Comment (lewtun) [inline .env.template:43]:
```suggestion
        "title": "Write a poem",
```

File: .env.template [hunk 4]
```diff
@@ -27,6 +27,30 @@ MODELS=`[
...
+        "prompt": "As a restaurant owner, write a professional email to the supplier to get these products every week: \n\n- Wine (x10)\n- Eggs (x24)\n- Bread (x12)"
```
Comment (lewtun) [inline .env.template:44]:
```suggestion
        "prompt": "Write a poem to help me remember the first 10 elements on the periodic table, giving each element its own line."
```

Comment (lewtun) [reply .env.template:44]: This is the output I get

In the realm of elements, a tale is told,                                                                                                                                                                 
Of atoms that make up our world, both new and old.                                                                                                                                                        
Their names and symbols, we shall now recite,                                                                                                                                                             
In a poetic form, to keep them in sight.                                                                                                                                                                  

Hydrogen, H, the lightest of them all,                                                                                                                                                                    
Found in water, it's the universe's call.                                                                                                                                                                 
Helium, He, the noble gas so fine,                                                                                                                                                                        
Floats balloons high, with its nature benign.                                                                                                                                                             

Lithium, Li, third in our elemental quest,                                                                                                                                                                
Light and reactive, in batteries it rests.                                                                                                                                                                
Beryllium, Be, a metal tough and strong,                                                                                                                                                                  
In alloys it shines, where it truly belongs.                                                                                                                                                              

Boron, B, a nonmetal with many a use,                                                                                                                                                                     
From glass to ceramics, its versatility loose.                                                                                                                                                            
Carbon, C, the backbone of life so dear,                                                                                                                                                                  
In diamonds and graphite, it's always near.                                                                                                                                                               

Nitrogen, N, the air we breathe and more,                                                                                                                                                                 
A crucial component, from plants to our core.                                                                                                                                                             
Oxygen, O, the gas that fuels our breath,                                                                                                                                                                 
Essential for life, till the hour of death.                                                                                                                                                               

Fluorine, F, a halogen, reactive and bold,                                                                                                                                                                
Found in toothpaste, its benefits untold.                                                                                                                                                                 
Neon, Ne, another noble gas to behold,                                                                                                                                                                    
In signs it glows, a sight to behold.                                                                                                                                                                     

These first ten elements, our world they shape,                                                                                                                                                           
From the smallest to the largest, their roles they drape.                                                                                                                                                 
So remember their symbols, their order and name,                                                                                                                                                          
In the periodic table, they'll always remain

File: .env.template (hunk 1 above)
Comment (Narsil) [reply .env.template:50]: This is an underdefined question.

Either Kayley is a male (or unidentified) and then she has 2 sisters like all her brothers.
Or Kayley is female and then she is a common sister of all her brothers, which means she has 1 sister.

Internet is wrong.
The dress is blue and black.

File: .env.template (hunk 4 above)
Comment (nsarrazin) [reply .env.template:44]: Oh I like this nice!

**Pull request merged**